.claude/positron-python.md

```diff
@@ -0,0 +1,151 @@
+# Positron Python Extension Development
+
+## Overview
+The Positron Python extension provides comprehensive Python language support and advanced data science features for Positron IDE, including interactive data exploration, kernel management, debugging, and environment detection. This extension is a fork of Microsoft's Python extension, enhanced for data science workflows.
+
+## Project Structure
+```
+extensions/positron-python/
+├── src/                           # TypeScript extension code
+├── python_files/                  # Python runtime components
+│   ├── posit/                    # Positron-specific Python modules
+│   │   ├── positron/             # Core Positron Python functionality
+│   │   │   └── tests/            # Python unit tests for Positron modules
+│   │   └── pyproject.toml        # Python project configuration
+│   └── lib/                      # Vendored Python dependencies
```

Comment:
We may want to include the `positron` test requirements file as well. I totally forgot which file is recommended - @isabelizimm do you remember?

Reply:
Will add